testsuite/integration-tests/src/test/java/org/jboss/intersmash/testsuite/provision/k8s/HyperfoilKubernetesOperatorProvisionerTest.java

```diff
@@ -0,0 +1,185 @@
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jboss.intersmash.testsuite.provision.k8s;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.Arrays;
+
+import org.jboss.intersmash.application.operator.HyperfoilOperatorApplication;
+import org.jboss.intersmash.junit5.IntersmashExtension;
+import org.jboss.intersmash.k8s.KubernetesConfig;
+import org.jboss.intersmash.k8s.client.Kuberneteses;
+import org.jboss.intersmash.provision.k8s.HyperfoilKubernetesOperatorProvisioner;
+import org.jboss.intersmash.provision.olm.OperatorGroup;
+import org.jboss.intersmash.provision.operator.HyperfoilOperatorProvisioner;
+import org.jboss.intersmash.provision.operator.hyperfoil.client.runschema.RunStatisticsWrapper;
+import org.jboss.intersmash.provision.operator.hyperfoil.client.v05.HyperfoilApi;
+import org.jboss.intersmash.provision.operator.hyperfoil.client.v05.invoker.ApiClient;
+import org.jboss.intersmash.provision.operator.hyperfoil.client.v05.invoker.ApiException;
+import org.jboss.intersmash.provision.operator.hyperfoil.client.v05.invoker.Configuration;
+import org.jboss.intersmash.provision.operator.hyperfoil.client.v05.model.Run;
+import org.jboss.intersmash.testsuite.junit5.categories.KubernetesTest;
+import org.jboss.intersmash.testsuite.k8s.NamespaceCreationCapable;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.MethodOrderer;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestMethodOrder;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import io.hyperfoil.v1alpha2.Hyperfoil;
+import io.hyperfoil.v1alpha2.HyperfoilBuilder;
+
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
+@KubernetesTest
+public class HyperfoilKubernetesOperatorProvisionerTest implements NamespaceCreationCapable {
+	private static final Logger logger = LoggerFactory.getLogger(HyperfoilKubernetesOperatorProvisionerTest.class);
+	private static final String NAME = "hyperfoil";
+	private static final HyperfoilKubernetesOperatorProvisioner hyperfoilOperatorProvisioner = initializeOperatorProvisioner();
+
+	private static HyperfoilKubernetesOperatorProvisioner initializeOperatorProvisioner() {
+		HyperfoilKubernetesOperatorProvisioner operatorProvisioner = new HyperfoilKubernetesOperatorProvisioner(
+				new HyperfoilOperatorApplication() {
+					@Override
+					public Hyperfoil getHyperfoil() {
+						Hyperfoil hyperfoil = new HyperfoilBuilder(
```

Comment:
this could return directly without using a local variable as in `HyperfoilOpenShiftOperatorProvisionerTest`

Reply:
Fixed.